### PR TITLE
fix: clipboard access(v3.9)

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* added selection range to clipboard to prevent IE access alert

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/js/common/clipboardUtil.js [hunk 1]
```diff
@@ -143,6 +143,26 @@ clipboardUtil = {
             return value.replace(LF, CUSTOM_LF_SUBCHAR)
                 .replace(CR, CUSTOM_CR_SUBCHAR);
         });
+    },
+
+    addClipboardSelection: function(el) {
```
Comment (seonim-ryu) [inline src/js/common/clipboardUtil.js]: 이 함수는 프라이빗이 아니라서 주석을 추가해야 되겠네요~

File: src/js/event/dragEventEmitter.js [hunk 2]
```diff
@@ -82,15 +82,15 @@ var DragEventEmitter = snippet.defineClass(/** @lends module:event/dragEventEmit
...
+     * @param {jQueryEvent} ev - Event object
      * @private
      */
     _onMouseMove: function(ev) {
         var gridEvent;
 
         // Prevent 'dragmove' from occuring when mouse button is not pressed.
         // This can happen when the alert dialog pops up from the the 'blur/mousedown' event handler.
-        if (!ev.buttons) {
+        if (!ev.which) {
```
Comment (seonim-ryu) [inline src/js/event/dragEventEmitter.js]: 제이쿼리 이벤트 객체라 `which` 사용이 가능하군요 굳아이디어 👍

File: src/js/common/clipboardUtil.js [hunk 3]
```diff
@@ -143,6 +143,26 @@ clipboardUtil = {
...
+        if (document.createRange) {
+            childNode = el.childNodes[0];
+            if (childNode) {
+                range = document.createRange();
+                selection = window.getSelection();
+                selection.removeAllRanges();
+                range.selectNodeContents(childNode);
+                selection.addRange(range);
+            }
+        // for IE8
```
Comment (seonim-ryu) [inline src/js/common/clipboardUtil.js]: `// for IE8` 이 주석은 아래 else 옆에 있어야할 것 같습니다만ㅎ
```js
} else { // // for IE8
...
```

File: src/js/common/clipboardUtil.js [hunk 4]
```diff
@@ -143,6 +143,26 @@ clipboardUtil = {
...
+        } else {
+            range = document.selection.createRange();
+            range.moveToElementText(el);
+            range.select();
+        }
     }
 };
```
Comment (seonim-ryu) [inline src/js/common/clipboardUtil.js]: 이 함수는 셀렉션을 풀었다가 다시 잡아주는 역할이라서..
네이밍이 `add-` 대신 `set-`으로 시작 하는게 어떨까요??
`setClipboardSelection`

File: src/js/common/clipboardUtil.js (hunk 1 above)
Comment (js87zz) [reply src/js/common/clipboardUtil.js]: 넵 추가하겠습니다~!

File: src/js/common/clipboardUtil.js (hunk 3 above)
Comment (js87zz) [reply src/js/common/clipboardUtil.js]: 변경했습니다~ㅋㅋ

File: src/js/common/clipboardUtil.js (hunk 4 above)
Comment (js87zz) [reply src/js/common/clipboardUtil.js]: 함수명 수정하였습니다~!

**Pull request merged**